build_tools/github_actions/runner/config/functions.sh

```diff
@@ -44,11 +44,20 @@ is_contained() {
   local match="$1"
   shift
   for e in "$@"; do
-    if [[ "$e" == "$match" ]]; then
+    if [[ "${e}" == "${match}" ]]; then
       return 0
     fi
   done
   return 1
 }
 
+# Retrieves the specified token to control the self-hosted runner.
+function get_token() {
+  local method=$1
+  local runner_scope=$2
+  local token_proxy_url="$(get_attribute github-token-proxy-url)"
+  local cloud_run_id_token=$(get_metadata "instance/service-accounts/default/identity?audience=${token_proxy_url}")
+  curl -sSfL "${token_proxy_url}/${method}" --header "Authorization: Bearer ${cloud_run_id_token}" --data-binary "{\"scope\": \"${runner_scope}\"}" | jq -r ".token"
```

Comment:
I know this is coming from something I suggested, so my fault, but I try to spell out command line options in scripts to make it clearer what's going on. IMO short options are for easy interactive usage. When checking something in, we can pay for the extra keystrokes. This is also a really long line. Let's break it up.

```suggestion
  curl --silent --fail --show-error --location \
   "${token_proxy_url}/${method}" \
   --header "Authorization: Bearer ${cloud_run_id_token}" \
   --data-binary "{\"scope\": \"${runner_scope}\"}" \
   | jq -r ".token"
```